.github/workflows/build-image.yml

```diff
@@ -0,0 +1,35 @@
+name: docker hub CI
+on:
+  push:
+    branches: master
```

Comment:
We push too often, change this to manual action only.

Reply:
I was thinking to put to tag the image with edge when comes from master branch, and only push latest image for tags in repo.
May it works like a manual action, some maintainer should tag a commit to push the latest image, what do you think?

Reply:
We commit a lot to master and have only a limited number of ci minutes, so that's why I don't think making a image for every master push is not worth it. 

This is how to set up a manual action: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

Reply:
done